packages/runtime/container-runtime/src/test/gc/gcConfigs.spec.ts

```diff
@@ -249,11 +253,34 @@ describe("Garbage Collection configurations", () => {
 				gcFeature: 1,
 				sessionExpiryTimeoutMs: customSessionExpiryDurationMs,
 				sweepTimeoutMs: 123,
-				gcFeatureMatrix: { tombstoneGeneration: 1, sweepGeneration: 1 },
+				gcFeatureMatrix: { tombstoneGeneration: 1, gcGeneration: 1 },
```

Comment:
Shouldn't `tombstoneGeneration`be removed to ensure that it's not going through the back-compat code? May be we can keep it in couple of back-compat specific tests.

Same comment for `gcTombstoneGenerationOptionName` option below.

Reply:
Yep, you're right.  I'll make a specific back-compat test and remove it otherwise.